src/lib/utils/index.js

```diff
@@ -0,0 +1,11 @@
+// Disabling because we intend to have more exports in the future.
+/* eslint-disable import/prefer-default-export */
+
+/**
+ * Checks whether a method exists in a given object
+ *
+ * @param  {String} methodName   The name of the method being checked.
```

Comment:
```suggestion
 * @param  {String} methodName   The name of the method being checked
```

Also, I don't really care, but I don't think we need more than two spaces between a variable name and its description (except when aligning, as below).